asv_bench/benchmarks/period.py

```diff
@@ -0,0 +1,9 @@
+from pandas import Period, PeriodIndex, date_range
+
+
+class create_period_index_from_date_range(object):
+    goal_time = 0.2
+
+    def time_period_index(self):
+        # Simulate irregular PeriodIndex
```

Comment:
So I'm not committing any new benchmarks, but let me run them locally before and after the change...